### PR TITLE
resolve page scroll when user selects last item

### LINKED ISSUE #231: page scrolls when user selects last item
Appears to be focus-related and can be resolved by placing focus on `a11y-focuser` prior to accessibility toggles made in `setStage`

Issue observed in Chrome & Edge, but not Firefox

### PR DESCRIPTION
**Pull request by chris-steele**
To resolve #231

### CODE REVIEW

**Comment by oliverfoster** [comment]
Could you try removing these instead please?

https://github.com/adaptlearning/adapt-contrib-narrative/blob/268cf26e4a1f8a67a6841b4cea922acfba300d6c/js/NarrativeView.js#L273-L274

**Review by oliverfoster — CHANGES_REQUESTED** [review]
See suggestion

**Comment by guywillis** [comment]
Both solutions resolve the page jump.

The suggestion from Ollie displays the previous button on the first slide and the next button on the last slide. Whilst selecting these buttons causes no errors within the build it is potentially confusing from a UI perspective and almost certainly confusing from a screen reader perspective - the buttons can be tabbed into and are read out.

Tested with NVDA.

**Comment by oliverfoster** [comment]
> Both solutions resolve the page jump.
> 
> The suggestion from Ollie displays the previous button on the first slide and the next button on the last slide. Whilst selecting these buttons causes no errors within the build it is potentially confusing from a UI perspective and almost certainly confusing from a screen reader perspective - the buttons can be tabbed into and are read out.
> 
> Tested with NVDA.

They should be read out as disabled.

**Comment by oliverfoster** [comment]
@guywillis Let us know about the nvda disabled bugs when you get a chance pls

**Comment by guywillis** [comment]
NVDA does not read the buttons as disabled. 

When testing, I have only removed these two lines as mentioned above:

> https://github.com/adaptlearning/adapt-contrib-narrative/blob/268cf26e4a1f8a67a6841b4cea922acfba300d6c/js/NarrativeView.js#L273-L274
> 

Screenshot of the narrative controls in the DOM for the first item of three.

Screenshot 2022-07-29 at 11 50 42

**Comment by oliverfoster** [comment]
@chris-steele can you replace 

https://github.com/adaptlearning/adapt-contrib-narrative/blob/268cf26e4a1f8a67a6841b4cea922acfba300d6c/js/NarrativeView.js#L273-L274

with 

```js
    a11y.toggleEnabled($left, !isAtStart);
    a11y.toggleEnabled($right, !isAtEnd);
```

instead?

**Comment by guywillis** [comment]
Works a charm.

Tested with NVDA

**Comment by chris-steele** [comment]
Done and thanks both

File: js/NarrativeView.js [hunk 1]
```diff
@@ -220,6 +220,8 @@ class NarrativeView extends ComponentView {
 
     this.$('.narrative__progress').removeClass('is-selected').filter(indexSelector).addClass('is-selected');
 
+    $('#a11y-focuser')[0].focus();
+
```
Comment (oliverfoster) [inline js/NarrativeView.js:224]:
```suggestion
```
Please remove

**Review by oliverfoster — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Pull request merged**